packages/angular/README.md

```diff
@@ -0,0 +1,116 @@
+# @tanstack-query-firebase/angular
+
+`@tanstack-query-firebase/angular` provides angular bindings for Firebase products.
+
+## Install Dependencies
+
+```bash
+npm i --save @angular/fire firebase @tanstack-query-firebase/angular
+```
+
+## Usage
+
+### Data Connect
+
+In your `app.config.ts`, add `provideTanstack` as a provider and `provideDataConnect` and `provideFirebaseApp`:
+
+```ts
+import { connectorConfig } from '@myorg/movies'; // Replace with your generated package name
+
+export const appConfig: ApplicationConfig = {
+  providers: [
+    ...
+    provideTanStackQuery(new QueryClient()),
+    provideFirebaseApp(() => initFirebase()),
```

Comment:
@maneesht , `@angular/fire/app` seems to export `initializeApp` , could `initFirebase` be a custom method? 

also, where is `provideDataConnect` imported from

Reply:
Updated for clarity.

Reply:
all good now, thanks!